packages/components/stamp/src/stamp.tsx

```diff
@@ -133,6 +151,17 @@ const getStampStyles = (props: StylesFunctionParams) => {
 const Stamp = (props: Props) => {
   const { themedValue } = useTheme();
   const overrideTextColor = themedValue(false, true);
+  const Icon =
+    props.icon &&
+    cloneElement(props.icon, {
+      size: 'medium',
+      color: getIconColor(props, overrideTextColor),
+    });
+
+  warning(
+    !props.children,
+    'Stamp: Please pass messages or icons as inline props, this method will be deprecated soon. For more information, see documentation: https://uikit.commercetools.com/?path=/story/components-stamps--stamp'
```

Comment:
```suggestion
    'Stamp: The `children` prop has been deprecated. Please use the `intlMessage` and `icon` prop to render the content.'
```

Reply:
Thank you.
updated here: https://github.com/commercetools/ui-kit/pull/2383/commits/18626634a4f71e9f068758455e03af90a99a8d9f